ucfcs/GrowthPlus
# Game updates and translation

This pr adds the timer to games 2, 3, 4 and fixes the responsiveness, sound volume, and bugs in the games. It also adds the translation to the results screen. 

TEST 
- run through the games, make sure that the timer works, you can hear the correct and incorrect sounds, and the animation is responsive (ex. the soccer ball goes in the net for the correct answer)
- Make sure that the points screen translates 


Closes #142 #188
